output/mistral_influxdb/readme.rst

```diff
@@ -6,6 +6,13 @@ database.
 
 The plug-in accepts the following command line options:
 
+--cert-path=certificate_path | -c certificate_path
+  The full path to a CA certificate used to sign the certificate of the InfluxDB server.
+
+--cert-dir=certificate_directory
+  The directory that contains the CA certificate(s) used to sign the certificate of the
```

Comment:
Probably something here to make clear that you don't need both cert-path and cert-dir